scripts/env_setup.sh

```diff
@@ -32,6 +51,9 @@ initializeEnvVars(){
 
 }
 
+#Get user bot token and store in json
+getBotToken
+
 #Get system level dependencies (python3, pip, etc.)
 getDependencies
```

Comment:
So like I said before, let's run the `getDependencies` function first and ensure `jq` is installed before we run `getBotToken`. In fact, let's run the functions in the following order:

```
#Get system and Server Duels dependencies
getDependencies

#Initialize the environmental variables
initializeEnvVars

#Get the bot token and store in config.json
getBotToken
```